__tests__/components/App.test.js

```diff
@@ -54,20 +52,23 @@ describe("#routes", () => {
       await expect(component.find(HomePage).length).toEqual(1)
     })
 
-    it('/editor renders Editor component', () => {
-      // Stub `Config.spoofSinopiaServer` static getter to force RT to come from server
-      jest.spyOn(Config, 'spoofSinopiaServer', 'get').mockReturnValue(false)
-      saveState({loginJwt: Config.awsCognitoJWTHashForTest, isAuthenticated: true}, 'jwtAuth')
-      const component = renderEditorRoute("/editor")
-      expect(component.find(ImportResourceTemplate).length).toEqual(1)
-    })
-
-    it('/templates renders the Login component (with location props) if the user is not authenticated', () => {
-      saveState({loginJwt: {}, isAuthenticated: false}, 'jwtAuth')
-      const component = renderRoutes("/templates")
-      expect(component.find(Login).length).toEqual(1)
-      expect(component.find(Login).prop('location')).toBeDefined()
-    })
+    //TODO: will need fixing
+    // it('/editor renders Editor component', () => {
```

Comment:
yeah, should definitely be a `.todo`, was an oversight on my part.  was going to leave as-is in this PR and deal with it under #528 (or whatever tickets it spawns), but will make it a `.todo` as part of this since i have to revisit a couple tests after rebasing anyway.

Reply:
turns out that you can only mark it `.todo` if there's no test body yet (i.e. no callback fn for the `it`).  so i marked these as `.skip`.  too bad, would've preferred `.todo`, but oh well.

https://jestjs.io/docs/en/api#testtodoname
> Note: If you supply a test callback function then the `test.todo` will throw an error. If you have already implemented the test and it is broken and you do not want it to run, then use `test.skip` instead.